codeserver/ubi9-python-3.11/run-code-server.sh

```diff
@@ -46,7 +46,9 @@ universal_json_settings='{
   "python.defaultInterpreterPath": "/opt/app-root/bin/python3",
   "telemetry.telemetryLevel": "off",
   "telemetry.enableTelemetry": false,
-  "workbench.enableExperiments": false
+  "workbench.enableExperiments": false,
```

Comment:
Thank you, Daniel, for opening this pull request.

I tested it using the generated image from the PR:
`quay.io/opendatahub/workbench-images:codeserver-ubi9-python-3.11-pr-739`

After comparing the old image with the one from this PR, I can confirm that the auto-update feature is now disabled by default. Users will need to manually enable it if desired.

Here’s a screenshot from the old image (Auto-update enabled by default):

![image](https://github.com/user-attachments/assets/57ef11f6-672b-4703-9b6d-102bef34a813)

And here’s the new image from this PR (Auto-update successfully disabled by default):

![image](https://github.com/user-attachments/assets/90b89508-8815-4723-9ccf-a47b9e20fd71)

/lgtm
/approve